### ISSUE TITLE
new-admin cannot be installed or compiled

### ISSUE THREAD

**Issue by matself** [hajkmap/Hajk#670]
**Describe the bug**
After some 20 minutes of extremely slow progress, at 100% cpu most of the time, npm fails with the following error. 
```
C:\Users\user\Hajk\new-admin>npm i
npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://npm.community>

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2021-02-11T18_18_43_002Z-debug.log
```
A subsequent attempt to start also fails, not unexpectedly
```
C:\Users\user\Hajk\new-admin>npm start
> hajk-admin@3.3.0 start C:\Users\user\Hajk\new-admin
> react-scripts start

react-scripts är inte ett internt kommando, externt kommando,
program eller kommandofil.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hajk-admin@3.3.0 start: `react-scripts start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hajk-admin@3.3.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2021-02-12T08_11_34_337Z-debug.log
```
An attempt to compile the component fails in very much the same fashion
```
C:\Users\user\Hajk\new-admin>npm run compile
npm ERR! missing script: compile

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2021-02-12T08_12_40_4
```

**To Reproduce**
Follow the Quick start instructions on this page.
https://github.com/hajkmap/Hajk

**Expected behavior**
As described in the instructions.

**Desktop (please complete the following information):**
 Windows 7, code fetched 210212

**Comment by matself** [comment]
I forgot to mention that new-backend and new-client can be installed and started according to the instructions. So there is nothing wrong with the basic machine setup

**Comment by jacobwod** [comment]
It looks clearly as an issue with your environment. I'm doing this dozens of times every day, as do other developers. Ensure you have recent NPM and Node versions.
```
~ % node -v
v12.20.1
npm -v
6.14.10
```

**Comment by sweco-sesqqq** [comment]
I just (some 2 minutes ago) built client and admin from master without any problems. 

It seems like: 
1) Installation of dependecies fails (not sure why, i would guess npm or node version-problems as Jacob said.)
2) Since you haven't installed the dependencies, react-scripts start cannot run
3) You try to run a "compile" script which we dont have in the package.json-file. (You should go with npm run build)

**Comment by matself** [comment]
Npm run build fails as well. And npm run compile is advised for new-backend. Any reason why new-admin behaves differently from new-client and new-backend?
I will check versions as suggested anyway.

**Comment by sweco-sesqqq** [comment]
Well, no surprise there since the dependencies are not installed. 

The reason for the diff between new-backend and new-admin/client (admin and client both uses "build") is simply that 
"build" is the standard name for compiling the solution when creating a new react-app (as opposed to an express-based app
where the name is compile instead).

I see no problem changing this terminology, but this is not the root of your problems. Can i close this? Feel free to contact me if you need more assistance.

**Comment by matself** [comment]
I have node v13.14.0 and npm 6.14.4
Should be no problem.
"It works for me" is never a helpful reply.

**Comment by jacobwod** [comment]
Mats, we've been trying to help you out here by supplying information on our setups and ensuring that there are no internal problems in Hajk itself. 

@sweco-sesqqq even checked out the project again - although he has a working setup already - with **the sole reason to reproduce the error you are getting**. It turns out that all dependencies install and all applications compiles as intended.

Furthermore, if you take a closer look at your logs it's clear that you have a problem with your tools:
```
npm ERR! This is an error with npm itself. 
```

Closing – could not reproduce.

**Comment by matself** [comment]
Google is your friend. The following broke whatever obstacle there was.
Clean npm cache `npm cache clean --force`
Delete the file package-lock.json
Empty the folder node_modules
re-run` npm install` and `npm start`
This gave dozens of deprecation warnings but eventually ended with a running development build.
This tip may be helpful in similar circumstances.
I will now proceed with build or compile commands.

**Comment by sweco-sesqqq** [comment]
Could you please attach the deprecated-warnings so i can take a look at them?

**Comment by matself** [comment]
Sure. You are lucky. I had not closed the cmd window.
[npm_output.txt](https://github.com/hajkmap/Hajk/files/5972066/npm_output.txt)

**Comment by matself** [comment]
I also noted that npm noted some vulnerabilities. Here they are.
C:\Users\user\Hajk\new-admin>npm audit
                       === npm audit security report ===
                                 Manual Review
             Some vulnerabilities require your attention to resolve
          Visit https://go.npm.me/audit-guide for additional guidance
  High            Cross-Site Scripting (XSS)
  Package         jquery
  Patched in      >=3.0.0
  Dependency of   jquery-sortable
  Path            jquery-sortable > jquery
  More info       https://npmjs.com/advisories/328
  Moderate        Prototype Pollution
  Package         jquery
  Patched in      >=3.4.0
  Dependency of   jquery-sortable
  Path            jquery-sortable > jquery
  More info       https://npmjs.com/advisories/796
  Moderate        Cross-Site Scripting
  Package         jquery
  Patched in      >=3.5.0
  Dependency of   jquery-sortable
  Path            jquery-sortable > jquery
  More info       https://npmjs.com/advisories/1518
found 3 vulnerabilities (2 moderate, 1 high) in 1781 scanned packages
  3 vulnerabilities require manual review. See the full report for details.
C:\Users\user\Hajk\new-admin>

**Comment by jussiraty** [comment]
3.6.0 release the "start": package.json contains a row  "start": "node dist/index.js". There is no such folder. I removed that dist/ and it started to work.
